src/movie/SearchBanner.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import './SearchBanner.scss';
+
+export function MovieSearchBanner(props) {
+  const [query, setQuery] = useState('');
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    props.onQueryChange(query);
+  };
+
+  return (
+    <div className="header-search-banner page-padding">
+      <Link to="/movie/0" className="btn btn-outline-danger btn-add-movie">
```

Comment:
why did you have static movie ID 0?? it's not correct

Reply:
I was using 0 to distinguish between add and edit movie(url- /movie/:id). In case of edit it would be actual movie id. However I will change the route as `movie/add` to better reflect the meaning.

Reply:
I was using 0 to distinguish between add and edit movie(url- /movie/:id). In case of edit it would be actual movie id. However I will change the route as `movie/add` to better reflect the meaning.